tests/unit/plugins/inventory/test_lxd.py

```diff
@@ -51,8 +51,14 @@ def inventory():
     return inv
 
 
+def test_verify_file(tmp_path, inventory):
+    file = tmp_path / "foobar.lxd.yml"
+    file.touch()
+    assert inventory.verify_file(str(file)) is True
+
+
 def test_verify_file_bad_config(inventory):
-    assert inventory.verify_file('foobar.lxd.yml') is False
+    assert inventory.verify_file('foobar.lsd.yml') is False
```

Comment:
Why this change?

Reply:
Thanks for the review. I tried to align the `test_verify_file_bad_config` test on all inventory plugins. at the moment some use typos, others don't. I'll remove this changes from this PR.